samples/javascript/package-lock.json

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
I don't know anything about JavaScript packages or npm, but are these lock files supposed to be checked-in?

Reply:
Convention is that we do check in package-lock.json files. Here's a [Medium](https://medium.com/@Quigley_Ja/everything-you-wanted-to-know-about-package-lock-json-b81911aa8ab8) article on the subject.